pkg/source/client/client.go

```diff
@@ -119,3 +119,12 @@ func NewProducer(ctx context.Context) (sarama.Client, error) {
 
 	return sarama.NewClient(bs, cfg)
 }
+
+// NewProducer is a helper method for constructing an admin client
+func MakeAdminClient(ctx context.Context, env *KafkaEnvConfig) (sarama.ClusterAdmin, error) {
```

Comment:
why not aligning with here: https://github.com/knative-sandbox/eventing-kafka/blob/ee7edfdd50d750db3b984c17f0d5cf835dcc3910/pkg/common/client/admin_client.go ?

Reply:
because the heavy-lifting is done in NewConfigWithEnv.